dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/pages/AllTypesPageNode.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.base.pages
+
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.pages.ContentNode
+import org.jetbrains.dokka.pages.ContentPage
+import org.jetbrains.dokka.pages.PageNode
+
+internal class AllTypesPageNode(
```

Comment:
Just in case people do find it and want to modify it

```suggestion
/**
 * This page is internal because it's an stdlib-specific feature,
 * which is not intended for public use or customization.
 * 
 * For more details, see https://github.com/Kotlin/dokka/issues/2887
 */
internal class AllTypesPageNode(
```